doc/adr/0007-api-endpoints-setup.md

```diff
@@ -0,0 +1,84 @@
+# 7. API endpoints setup
+
+Date: 2024-03-07
```

Comment:
Just curious: Was that the date where we actually agreed?

Reply:
It was the date where the ADR was written because I forgot when we argeed :D